aws/monolith-terraform/xebialabs/README.md.tmpl

```diff
@@ -0,0 +1,27 @@
+{{$app := .AppName | kebabcase}}
+## Monolith application on AWS ECS with Fargate
+
+The blueprint deploys a monolithic application, generated with JHipster, to AWS ECS with the Fargate launch type.
+XL Deploy does the provisioning and deployment, while XL Release orchestrates everything.
+
+The application is available in the repo [https://github.com/xebialabs/e-commerce-monolith](https://github.com/xebialabs/e-commerce-monolith).
+
+### Prerequisites
+
+If you do not have XL Release and XL Deploy instances running, you can use this Docker Compose setup to spin them up. The setup also contains a preconfigured Jenkins instance.
```

Comment:
we have to mention that terraform installation ins required on the docker container